templates/security/notices.html

```diff
@@ -6,9 +6,13 @@
 <section class="p-strip--suru-topped">
   <div class="row">
     <div class="col-8">
-	    <h1>Ubuntu Security Notices</h1>
-      <p>Developers issue an Ubuntu Security Notice when a security issue is fixed in an <a href="#">official Ubuntu package</a>.</p>
-      <p>To report a security vulnerability in an Ubuntu package, please <a href="#">contact the Security Team</a>.</p>
+      {% if request.args.get('release') %}
```

Comment:
`{% if request.args.get('release') or  request.args.get('details') %}` right?

Reply:
Yeah I did start with that check initially but, with all the checks throughout the template, it got quite noisy, and I can't see that there'd be an instance where both aren't present (I could be wrong).

Could we maybe pass something to the view from views.py to indicate it's returning search results?